src/sema/eval.rs

```diff
@@ -196,3 +196,324 @@ pub fn eval_const_rational(
         )),
     }
 }
+
+/// Function that recurses the expression, and calls a function that folds number literals in the expression(eval_constants_in_expression())
```

Comment:
What does this mean?
`expression(eval_constants_in_expression())`

Reply:
I meant to put the name of the function that folds the numberliterals

Reply:
> I meant to put the name of the function that folds the numberliterals

I think you can rewrite that. `expression(eval_constants_in_expression())` looks like `expression` is a function.